src/main/resources/application.yaml

```diff
@@ -67,36 +67,23 @@ azure:
   application-insights:
     instrumentation-key: ${ccd.AppInsightsInstrumentationKey:00000000-0000-0000-0000-000000000000}
 
-testS2S:
-  api:
-    url: "/testS2SAuthorization"
-integrationTest:
-  api:
-    url: "/"
-
 feign:
   client:
     config:
       default:
         connectTimeout: 5000
         readTimeout: 5000
-        loggerLevel: basic
   allow:
     headers:
       - authorization
       - serviceauthorization
       - user-roles
       - user-id
-ccd:
-  case-definition:
-    host: ${DEFINITION_STORE_HOST:http://localhost:4451}
 
 documentTTL: ${DOCUMENT_TTL:3600000}
 
 documentStoreUrl: ${DM_STORE_BASE_URL:http://localhost:4506}
 caseDataStoreUrl: ${CCD_DATA_STORE_API_BASE_URL:http://localhost:4452}
-swaggerUrl: ${CASE_DOCUMENT_SWAGGER_AM_URL:localhost:4455}
-caseDocumentAmUrl: ${TEST_URL:http://localhost:4455}
```

Comment:
You may wish to update https://github.com/hmcts/ccd-docker/blob/master/compose/case-document-am.yml to match

Reply:
yes, thanks Matt. Fixed in 44d1567116b37283c5fc977c5fdc42a081aa428f